src/editortoolkit_neume.cpp

```diff
@@ -699,8 +699,7 @@ bool EditorToolkitNeume::Insert(std::string elementType, std::string staffId, in
         ClassIdComparison ac(STAFF);
         parent->FindAllChildByComparison(&staves, &ac);
         staves.push_back(newStaff);
-        StaffSort staffSort;
-        std::stable_sort(staves.begin(), staves.end(), staffSort);
+        std::stable_sort(staves.begin(), staves.end(), Object::sortByUlx);
```

Comment:
We need to figure out where in reading order this appears. `Object::sortByUlx` would not do this, but staffSort does need to be updated to deal with skews.